### PR TITLE
Make static space turfs dark

### PR DESCRIPTION
**Pull request by ghost**
I mean come on, it's 2016

The background in all static space turfs (transit turfs would be hell on earth to work through, would need to figure out what the fuck they did to render those and redo it for the new turfs) is now pitch black instead of dark gray (21, 21, 21)

Pretty small change, but it's time we start cleaning up oldcoder legacy

spess2

### CODE REVIEW

**Comment by ComicIronic** [comment]
That sure is a nice sprite PR you have with no comparison images.

**Comment by PJB3005** [comment]
Check the space parallax PR for an example of what it looks like.

**Comment by Probe1** [comment]
image

kinda hard to see the diff

**Comment by ihadtoregisterforthis** [comment]
000000 space blended with the color of hidden turfs on the parallax testing IIRC.

**Comment by PJB3005** [comment]
@Probe1 should've taken an in game picture of space instead.

**Comment by 9600bauds** [comment]
Can't you just use Photoshop background eraser on low tolerance for the transit turfs?

**Comment by ghost** [comment]
It's a bit more complicated than that, there's low luminosity differences in the transit turfs so I'm pretty sure the only way to really do it is regenerate them

**Comment by ComicIronic** [comment]
Yeah this needs game images for comparison

**Comment by ghost** [comment]
I'll give it a check momentarily

**Comment by ghost** [comment]
spess

Could use some blank tiles, I'll give it a look in a moment, but it certainly looks a lot more sharp to me instead of the old muddy space

**Comment by Kurfursten** [comment]
If I recall correctly from the test server, pitch black does look nice but it also makes it annoying when you don't have LOS to something because the black suddenly blends with the black of space

**Comment by ghost** [comment]
I decided to go for a more pragmatic solution and thinned the space tiles a wee bit. I will do a first test run and give the screenshot, and see if I need to thin it further

**Comment by ghost** [comment]
Okay, not even close to thin enough, back to the drawing board

**Comment by ghost** [comment]
spess2

Lot less stars, also more color mix instead of just 70 % white and 20 % blue

**Comment by ComicIronic** [comment]
Needs a shot with darkness as well.

**Comment by dylanstrategie** [comment]
Que ?

**Comment by ghost** [comment]
Now that I took a look at it, it looks a bit odd

space3

Two solutions here. Keep as is, or crank down space luminosity a bit

File: data/investigate/singulo.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<small>21:50 [0x201bbd2] (238,402,4)</small> || Energy Cannon regained power and turned <font color='green'>on</font><br />
```
Comment (PJB3005) [inline data/investigate/singulo.html]: Blew.

File: data/investigate/singulo.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+<small>21:50 [0x201bbc7] (242,403,4)</small> || Energy Cannon regained power and turned <font color='green'>on</font><br />
```
Comment (PJB3005) [inline data/investigate/singulo.html]: It.

**Comment by ghost** [comment]
Okay, I'll just keep it as is

Let's just say, to put it simply, that fixing this would need some pretty heavy-handed lighting engine reworks

**Comment by clusterfack** [comment]
We should change what darkness itself looks like (yes you can do this).

**Comment by invoid9** [comment]
Adjust the angle of your monitor. The difference is awesome. SPACE IS BLACK AND SHOULD BE BLACK THIS IS THE RIGHT THING TO DO FOR THE FUTURE OF SPESSMAN

**Comment by clusterfack** [comment]
A nice pink darkness sounds nice doesn't it?

**Comment by 9600bauds** [comment]
Uhhhhh

**Comment by ComicIronic** [comment]
Make darkness (21,21,21)

**Comment by ghost** [comment]
Would still be needed, but I think it should be integrated in the new parallax system. AKA use the method for generating a parallax layer to generate a static space background for people who have toasters or just don't want to see shit moving

**Pull request closed without merging**